pom.xml

```diff
@@ -274,6 +274,7 @@
                     <artifactId>maven-surefire-plugin</artifactId>
                     <configuration>
                         <argLine>@{jacocoAgent} @{surefire.argLine}</argLine>
+                        <parallel>all</parallel>
```

Comment:
Doesn't work with junit Jupiter. Instead, use their properties in argLine or resource property files.

The latter will be picked up by IDEs

Reply:
done